api.json

```diff
@@ -0,0 +1,761 @@
+{
+  "title": "twine-api",
+  "version": "0.1",
+  "routes": {
+    "/sectors": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "Retrieve all users accessible with the given scopes",
+          "auth": {},
+          "scope": {},
+          "query": {},
+          "response": {}
+        }
+      },
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
+              "id": "Number",
+              "name": "String",
+              "logo_url": "String",
+              "address1": "String",
+              "address2": "String",
+              "townCity": "String",
+              "postCode": "String",
+              "coordinates": "String",
+              "region": "String",
+              "sector": "String",
+              "registeredAt": "String",
+              "lastActiveAt": "String",
+              "modifiedAt": "String",
+              "turnoverBand": "String",
+              "360GivingId": "String",
+              "fundingProgramme": "String",
+              "fundingBody": "String",
+              "parentOrganisation": "String",
+              "childOrganisations": ["String"],
+              "subscription": "String",
+              "trainingSessions": [{
+                "trainingSessionAttendees": ["Number"],
+                "trainingSessionNotes": "String",
+                "eventDate": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/regions": {
+      "description": "",
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
+              "id": "Number",
+              "name": "String",
+              "logo_url": "String",
+              "address1": "String",
+              "address2": "String",
+              "townCity": "String",
+              "postCode": "String",
+              "coordinates": "String",
+              "region": "String",
+              "sector": "String",
+              "registeredAt": "String",
+              "lastActiveAt": "String",
+              "modifiedAt": "String",
+              "turnoverBand": "String",
+              "360GivingId": "String",
+              "fundingProgramme": "String",
+              "fundingBody": "String",
+              "parentOrganisation": "String",
+              "childOrganisations": ["String"],
```

Comment:
Including this kind of information requires additional queries. I thought about making it an opt-in field by default, but I guess that would complicate the backed logic a fair bit, and we control all the clients anyway so it's probably not that big of a deal.

TLDR: this is fine, just wanted to add this note.

Reply:
Yeah, makes sense. Also `parentOrganisation` can probably be dropped now as it's the same as `fundingBody`